app/controllers/admin/elections_controller.rb

```diff
@@ -21,11 +21,9 @@ def create
     @election = Election.new(election_params)
     respond_to do |format|
       if @election.save
-        format.html { redirect_to @election, :notice => 'Eventet skapades!' }
-        format.json { render :json => @election, :status => :created, :location => @election }
+        format.html { redirect_to [:admin,@election], notice: 'Valet skapades!' }
       else
-        format.html { render :action => "new" }
-        format.json { render :json => @election.errors, :status => :unprocessable_entity }
+        format.html { render action: "new" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.